testutil/base.go

```diff
@@ -60,3 +61,32 @@ func (s *BaseTest) TearDownTest(c *check.C) {
 func (s *BaseTest) AddCleanup(f func()) {
 	s.cleanupHandlers = append(s.cleanupHandlers, f)
 }
+
+// BackupBeforeMocking backups the specified list of elements before further mocking
+func BackupBeforeMocking(mockablesByPtr ...interface{}) (restore func()) {
```

Comment:
given that the package is testing-focused, this can be called just Backup I think.